### DISCUSSION TITLE
OpenBSD support

### DISCUSSION THREAD

**Discussion by ghost** [NVIDIA/open-gpu-kernel-modules#325] [Q&A]
Will there be a driver for OpenBSD?

**Comment by YusufKhan-gamedev** [comment]
Ah, I dont know the status on that but...that would be a maintenance nightmare :rofl: 

Oh, and also OpenBSD doesnt support kernel modules

**Comment by niv** [comment]
Hi! Thank you for your interest. This repository is just for Linux. We have no plans to add BSD support to this package.
